offchain-modules/packages/x/src/handlers/ckb.ts

```diff
@@ -761,6 +771,36 @@ export class CkbHandler {
     });
   }
 
+  async collectUnlockSignatures(
+    txSkeleton: TransactionSkeletonType,
+    records: ICkbUnlock[],
+    generator: CkbTxGenerator,
+  ): Promise<string[]> {
+    const trickyLumosConfig = {
+      PREFIX: generator.lumosConfig.PREFIX,
+      SCRIPTS: {
+        SECP256K1_BLAKE160_MULTISIG: generator.lumosConfig.SCRIPTS.SECP256K1_BLAKE160_MULTISIG,
+      },
+    };
+    txSkeleton = secp256k1Blake160Multisig.prepareSigningEntries(txSkeleton, {
+      config: trickyLumosConfig,
+    });
```

Comment:
No need to prepareSigningEntries again, already done in `fn generator.unlock()`

Reply:
fixed, reviewed generator unlock and understood prepareSigningEntries function